### PR TITLE
[BugFix] Auth token expiry and date conversion

### PR DESCRIPTION
**Pull request by Ram81**
### Description

This PR - 

- [x] Fix date conversion on profile page
- [x] Handle exception in token refresh when token is expired 
- [x] Show token expiry on refresh token modal

### CODE REVIEW

File: apps/accounts/models.py [hunk 1]
```diff
@@ -64,6 +64,7 @@ class JwtToken(TimeStampedModel):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     access_token = models.CharField(max_length=512, blank=False, null=True)
     refresh_token = models.CharField(max_length=512, blank=False, null=True)
+    expires_at = models.PositiveIntegerField(default=86400)
```
Comment (RishabhJain2018) [inline apps/accounts/models.py:67]: Shouldn't it be a date field?

Comment (RishabhJain2018) [reply apps/accounts/models.py:67]: Also, why do we need this field? Can't we calculate it dynamically?

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
Please fix the travis build as well.

File: apps/accounts/models.py (hunk 1 above)
Comment (Ram81) [reply apps/accounts/models.py:67]: Because it is epoch time and not a date time field. Replied to second comment on channel

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**